### ISSUE TITLE
Any chances to get Shellcheck onboard ?

### ISSUE THREAD

**Issue by netflash** [singapore/lint-condo#277]
Hi there,

any chances you can get https://github.com/koalaman/shellcheck onboard ?
Would love to have it to check bash scripts.

**Comment by rarkins** [comment]
Yes, no problem - I'll add shortly or feel free to submit a PR in the meantime

**Comment by rarkins** [comment]
After some quick reading, it seems that `shellcheck` isn't as simple as some other linters.

It's not supported directly by alpine linux: https://bugs.alpinelinux.org/issues/6417

Here's an example docker image that incorporates alpine + shellcheck: https://github.com/NLKNguyen/alpine-shellcheck

The only problem is that it involves manual steps, while this repo is auto-built by Docker Hub. It could still be possible but needs more thought.

**Comment by rarkins** [comment]
One (ugly) possibility would be to simply commit the shellcheck binary to this repo and copy it into the Docker image.

**Comment by netflash** [comment]
Nice findings!
I find Docker Hub and Quay native build processes not good for `separate build container` approach :-(

Thought of using releases https://github.com/koalaman/shellcheck/releases , but there's source code only...

**Comment by damiendurant** [comment]
Hi,

I compile the shellcheck on this fork : https://github.com/damiendurant/lint-condo.git

It's based on work done on mhart/alpine-node
Therefore, currently the image is 600M ,still looking to downsize it before to merge request.
